### ISSUE TITLE
KMLDataSource TypeError: Cannot read property 'scene' of undefined

### ISSUE THREAD

**Issue by ksrb** [darwin-education/resium#10]
I'm pretty sure you're aware of this issue as it's broken in storybook but attempting to use the `<KMLDataSource>` just throws:
```bash
TypeError: Cannot read property 'scene' of undefined
```

Example use:
```javascript
render() {
  return (
    <div>
      <KMLDataSource />
    </div>
  )
}
```

I've hacked around it in https://github.com/rot1024/cesium-react/blob/ac18cc9efe01fc3b4698c18a25014bd4b7e80b24/src/KmlDataSource.js#L33-L40 by doing something along the lines of:
```javascript
  createCesiumElement(options) {
    let scene = null;
    if (this.context) {
      scene = this.context.scene;
    }
    return new CesiumKmlDataSource({
      camera: options.camera || (scene ? scene.camera : undefined),
      canvas: options.canvas || (scene ? scene.canvas : undefined),
      proxy: options.proxy,
    });
  }
```

Clearly this is a bit of a life cycle issue but I'm just wondering how you would go about fixing this... would passing the camera and canvas options to the `<Viewer>` and `<KMLDataSource>` be something you had considered? Obviously not ideal just trying to understand the original intent.

**Comment by rot1024** [comment]
This issue will be fixed in v1.0.0

**Comment by rot1024** [comment]
Hi, I've released v1.0.0 and fixed this bug. Please try it!
